crates/dscvr-telemetry-util/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+use axum::{extract::MatchedPath, middleware::Next, response::Response};
+use http::Request;
+use std::time::Instant;
+
+pub async fn track_metrics<B>(req: Request<B>, next: Next<B>) -> Response {
```

Comment:
wdyt of calling this `axum_prometheus_track_metrics`? Or better yet, have a module called `axum`, and have this in there

Reply:
@jeffgrunewald we forgot to discuss this, but since this is a generic telemetry-util crate, might be worth placing this in a `axum` module. Alternately, rename the crate to dscvr-axum-util